concourse/scripts/kibana_set_utc.rb

```diff
@@ -0,0 +1,43 @@
+#! /usr/bin/env ruby
+
+require 'net/http'
+require 'json'
+
+def ingest_to_es(document, uri)
+  raise "Document is not a hash" unless document.is_a? Hash
+  Net::HTTP.new(uri.host, uri.port).start do |http|
+    req = Net::HTTP::Put.new(uri.path, 'Content-Type' => 'application/json')
+    req.body = document.to_json
+    http.request(req)
+  end
+end
+
+def date_format_not_utc(response_json)
+end
+
+es_host = ENV.fetch("ES_HOST").delete(' ')
+es_port = ENV.fetch("ES_PORT").delete(' ')
+
+es_url = "http://#{es_host}:#{es_port}"
+date_format_url = "#{es_url}/.kibana/config/4.3.1"
+date_format_uri = URI(date_format_url)
+
+response = JSON.parse(Net::HTTP.get(date_format_uri))
+
+if ! response["found"] || response["_source"]["dateFormat:tz"] != "UTC"
+  index_settings_url = es_url + '/.kibana'
+  index_settings_uri = URI(index_settings_url)
+  index_settings = {
+    "settings" =>
+    { "index" => {
+      "number_of_shards" => 1,
+      "number_of_replicas" => 1,
```

Comment:
It might also be clearer to break it out to two separate conditions:
```ruby
if ! response["found"]
  ingest_to_es(index_settings, index_settings_uri)
end

if response["_source"]["dateFormat:tz"] != "UTC"
  ingest_to_es(timezone_settings, date_format_uri)
end
```